source/cl/test/UnitCL/source/clEnqueueNDRangeKernel.cpp

```diff
@@ -526,7 +526,7 @@ TEST_F(clEnqueueNDRangeKernelTest, ConcurrentContextCreateAndEnqueue) {
         clCreateContext(nullptr, 1, &device, nullptr, nullptr, nullptr);
     cl_command_queue queue = clCreateCommandQueue(ctx, device, 0, nullptr);
 
-    std::vector<cl_event> kernel_complete_events;
+    std::vector<cl_event> kernel_complete_events{};
```

Comment:
I think this has the same effect? We don't do this elsewhere in UnitCL.

Reply:
You reverted one `std::vector` change in response to Colin's comment, but not the one he commented on, so I think this was resolved too soon. There is also another `std::vector` later in this file that did not need updating. (Mind you, it does the same thing, and if you had responded that you would prefer to keep them in for some reason, I'd also be fine with that.)

Reply:
Done.